app/app.go

```diff
@@ -338,7 +338,7 @@ func NewNoisApp(
 	if err != nil {
 		panic(fmt.Sprintf("error while reading wasm config: %s", err))
 	}
-	availableCapabilities := "iterator,staking,stargate,stargaze,cosmwasm_1_1"
+	availableCapabilities := "iterator,staking,stargate,cosmwasm_1_1"
```

Comment:
Nice one. I pulled this out into https://github.com/noislabs/noisd/pull/13 to add cosmwasm_1_2